### DISCUSSION TITLE
在英伟达G-SYNC下全屏无图像

### DISCUSSION THREAD

**Discussion by SEKAI-NO-YAMADA** [ywmoyue/biliuwp-lite#697] [General]
### Bug 描述

开启英伟达G-SYNC后，在视频详情页有图像和音频，全屏后无图像，但进度条和音频都在走
关闭此功能后，正常

### 操作步骤

```plain text
开启英伟达G-SYNC
任意方式进入任意视频
进入全屏，无图像
关闭英伟达G-SYNC后，正常
```

### 期望结果

不搞也无所谓

### 截图

<img width="1273" alt="视频详情页" src="https://github.com/ywmoyue/biliuwp-lite/assets/174694466/0c830a0b-cdbf-4623-bd0d-f47840f4a705">
<img width="416" alt="进入全屏后" src="https://github.com/ywmoyue/biliuwp-lite/assets/174694466/21118116-a3cf-45a2-ac98-c0f3eaf053e0">

### 应用来源

Github/其他Github镜像站

### 系统版本

Windows 11 21H2 (Build 22000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/c37d37dd-5d75-4dcf-9bd5-6a4316d57db4)

@SEKAI-NO-YAMADA 试试播放设置中画面尺寸设置为兼容

**Reply by SEKAI-NO-YAMADA** [reply]
好的，我换了台电脑也没有复现，可能是电脑出了点小bug
现在好了，麻烦了

**Comment by ywmoyue** [comment]
我试了下启用GSync后也没有复现，不清楚是否还有其他因素影响
